doc/community/contrib-snip.rst

```diff
@@ -9,11 +9,12 @@ send an email to falcon@librelist.com and follow the instructions in the
 reply. For more information about managing your subscription, check out
 the `Librelist help page <http://librelist.com/help.html>`_.
 
-While we don't have an official code of conduct, we do expect everyone
-who participates on the mailing list to act professionally, and lead
-by example in encouraging constructive discussions. Each individual in
-the community is responsible for creating a positive, constructive, and
-productive culture.
+All contributors and maintainers of this project are subject to our `code
+of conduct <https://github.com/falconry/falcon/blob/master/CODEOFCONDUCT.md>`_.
```

Comment:
"Code of Conduct" (caps)

Its a thing, an important thing, so should probably be caps.